common/configs/src/eth.rs

```diff
@@ -27,13 +27,14 @@ use serfig::parsers::Toml;
 use crate::LogConfig;
 use crate::StorageConfig;
 
-#[derive(Parser, Debug, Default, Clone, Serialize, Deserialize)]
+#[derive(Parser, Debug, Clone, Serialize, Deserialize)]
 pub struct ExportConfig {
     #[clap(
         short = 'p',
         long,
         value_parser,
-        default_value = "https://127.0.0.1:8845",
+        default_value_t,
+        default_value = "127.0.0.1:8845",
```

Comment:
```suggestion
        default_value = "https://127.0.0.1:8845",
```

Should be exactly the same with Default export config.